cypress/e2e/webTables.cy.js

```diff
@@ -1,7 +1,101 @@
 /// <reference types='cypress' />
 
-describe('Web Tables page', () => {
-  it('', () => {
+describe('Web Tables - Pagination Test', () => {
+  it('should change the page size and verify the number of rows', () => {
+    cy.visit('https://demoqa.com/webtables');
 
+    cy.get('select[aria-label="rows per page"]').select('20');
+
+    cy.get('.rt-tbody .rt-tr-group').should('have.length', 20);
+
+    cy.get('select[aria-label="rows per page"]').select('50');
+
+    cy.get('.rt-tbody .rt-tr-group').should('have.length', 50);
+
+    cy.get('select[aria-label="rows per page"]').select('100');
+
+    cy.get('.rt-tbody .rt-tr-group').should('have.length', 100);
+  });
+});
+
+
+
+
+describe('Web Tables - Rows Count Selection', () => {
+  it('should change the number of displayed rows', () => {
+    cy.visit('https://demoqa.com/webtables');
+
+    cy.get('select[aria-label="rows per page"]').select('10');
+
+    cy.get('.rt-tbody .rt-tr-group').should('have.length', 10);
+  });
+
+});
+describe('Web Tables - Add New Worker', () => {
+  it('should add a new worker successfully', () => {
+    cy.visit('https://demoqa.com/webtables');
+
+    cy.get('#addNewRecordButton').click();
+
+    cy.get('#firstName').type('John');
+    cy.get('#lastName').type('Doe');
+    cy.get('#userEmail').type('john.doe@example.com');
+    cy.get('#age').type('30');
+    cy.get('#salary').type('50000');
+    cy.get('#department').type('Engineering');
+
+    cy.get('#submit').click();
+
+    cy.get('.rt-tbody').should('contain.text', 'John');
+    cy.get('.rt-tbody').should('contain.text', 'Doe');
   });
 });
+describe('Web Tables - Delete Worker', () => {
+  it('should delete a worker successfully', () => {
```

Comment:
if you deleted Cierra once they will not be displayed in the second test running so the test will be failed. This means we should create workers first and then delete them (you can make a custom command for the worker creation)

Reply:
Fixed